metrics/web/docs/features/remote_config/diagrams/remote_config_data_layer_class_diagram.puml

```diff
@@ -0,0 +1,26 @@
+@startuml remote_config_data_layer_class_diagram
+
+package common.domain.repository {
+    interface RemoteConfigurationRepository {}
+}
+
+package common.domain.entities {
+    interface RemoteConfiguration {}
+}
+
+package common.data {
+    package repository {
+        class FirebaseRemoteConfigurationRepository {}
+    }
+
+    package model {
+        class RemoteConfigurationData {}
```

Comment:
Do we need a `Deserializer` here for the `RemoteConfiguration` entity? As the remote config doesn't return a usual JSON and uses the `Map<String, RemoteConfigValue>` instead, it seems like a `RemoteConfigurationDeserializer` should be enough as we don't need the `DataModel` here with `toJson()` implementation. See the sequence diagram [here](https://github.com/platform-platform/monorepo/blob/master/metrics/web/docs/01_metrics_web_application_architecture.md#communication-between-layers).